### ISSUE TITLE
Quick Toolbars plugin > Quick Image bugs from version 6.4.0

### ISSUE THREAD

**Issue by yim-jj** [tinymce/tinymce#8626]
**What is the current behavior? Describe the bug**
When inserting an image by clicking the Quick Image tollbar button, only the first image is inserted and subsequent images are not inserted.

**Please provide the steps to reproduce and if possible a minimal demo of the problem via [fiddle.tiny.cloud](https://fiddle.tiny.cloud/) or similar.**
```
{
...
plugins: 'quickbars',
toolbar: 'quickimage'
...
}
```

**What is the expected behavior?**
Should be inserted whenever an image is entered. Regardless of whether it is the same image or a different image

**Which versions of TinyMCE, and which browser / OS are affected by this issue? Did this work in previous versions of TinyMCE?**
- From TinyMCE version 6.4.0
- Chrome browser
- macOS & Windows

However, it is normal until version 6.3.2

**Comment by TheSpyder** [comment]
Our QA team has confirmed this only impacts Chrome, not Firefox or Safari. So while it is still a regression there is a workaround.

**Comment by higty** [comment]
I just meet same bug.
I had work around by specify 6.3.
```
<script src="https://cdn.tiny.cloud/1/3a6lsu04zxdoo7oc2dfg98ptzn3cz8wk7gtdqt16jr0y4pxw/tinymce/6.3/tinymce.min.js" referrerpolicy="origin"></script>
```

**Comment by huyanyawei** [comment]
@TheSpyder  Please be professional, you need to know that the real use of rich text is the operator, they use Google, and there have been too many feedbacks recently, causing the boss to ask us to change to other

**Comment by TheSpyder** [comment]
Fixed by #8678, we should be releasing that in version 6.4.2 shortly
